timeline/templates/timeline/timeline_site.html

```diff
@@ -1,106 +0,0 @@
-{% extends 'projectroles/base.html' %}
```

Comment:
This should not be deleted, instead it should form the basis for both the site-wide events and the admin view.